oracle/pyproject.toml

```diff
@@ -40,10 +40,10 @@ license = "BSD-3-Clause"
 deps = [
     "jaydebeapi==1.2.3",
     "jpype1==1.4.1; python_version > '3.0'",
-    "oracledb==1.3.2; python_version >= '3.8'",
+    "oracledb==1.4.0; python_version >= '3.8'",
```

Comment:
Nonblocking nit but we should change `python_version >= '3.8'` to `python_version > '3.0'` to be consistent with most other dependencies.

Reply:
yup it will be easier to drop them all at once once we drop py2, you're right!